### ISSUE TITLE
Update the documentation

### ISSUE THREAD

**Issue by Moppa5** [Seravo/seravo-plugin#557]
**Description**
Currently the README.md has a lots of stuff there about Seravo Plugin. However, some of that don't seem to be too up to date anymore. As a part of the Seravo Plugin 2.0 project we should discuss, plan, and maybe even rewrite the whole documentation.

**Comment by JoosuaKoskinen** [comment]
Agreed. We currently have README.md and CONTRIBUTING.md. README.md should be updated to only have the most relevant information. I would also like to have DEVELOPERS.md with instructions on developing Seravo Plugin and information about Seravo Plugin hooks etc which site developers need to know.
